.circleci/config.yml

```diff
@@ -1249,7 +1253,7 @@ workflows:
       - fe-tests-cypress:
           matrix:
             parameters:
-              edition: ["ee", "oss"]
+              edition: ["ee"]
```

Comment:
Shouldn't we build only OSS according to https://github.com/metabase/metabase/pull/23123/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R1293 ??

Reply:
Totally got lost regarding the `fe-tests-cypress` names.